src/components/UserProfile/UserProfile.jsx

```diff
@@ -301,6 +301,7 @@ const UserProfile = props => {
     }
     setShouldRefresh(true);
     setChanged(false);
+    window.location.reload();
```

Comment:
Tasks are still showing after deleting them. We have to update the state of the Project task dropdown too.
I think  : )